app/controllers/clearance/sessions_controller.rb

```diff
@@ -33,4 +40,8 @@ def url_after_create
   def url_after_destroy
     sign_in_url
   end
+
+  def url_for_signed_in_users
+    url_after_create
```

Comment:
Would it be possible to make this a configuration value like `url_after_create` does? Now that there are multiple types of redirects, seems like `Clearance.configuration.redirect_url` should be renamed to something like `Clearance.configuration.after_create_redirect_url` and add a `Clearance.configuration.after_sign_in_redirect_url`